README.md

```diff
@@ -1,75 +1,46 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse!
 
-> One paragraph statement about the project.
+> A program sending greeings to microverse
+![screenshot](./screenshot.png)
 
-![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
+## Built With
 
-### Prerequisites
+- HTML
+- CSS
 
-### Setup
```

Comment:
- [ ] Ensure to add steps on how users can get a copy of your project locally - like cloning or download.

- [ ] All your linters are not green indicating that some are failing passing. Please ensure to correct your linter errors so that all of them are passing

- [ ] Your Pull Request title is not particularly descriptive of what features (or project) you want to merge. Please change **'Feature a'** to something more descriptive of this project